src/main/scala/com/microsoft/partnercatalyst/fortis/spark/pipeline/TwitterPipeline.scala

```diff
@@ -0,0 +1,51 @@
+package com.microsoft.partnercatalyst.fortis.spark.pipeline
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.{Analysis, AnalyzedItem, FortisItem}
+import com.microsoft.partnercatalyst.fortis.spark.streamprovider.{ConnectorConfig, StreamProvider}
+import org.apache.spark.streaming.StreamingContext
+import org.apache.spark.streaming.dstream.DStream
+import twitter4j.{Status => TwitterStatus}
+
+object TwitterPipeline extends Pipeline {
+  override def apply(streamProvider: StreamProvider, streamRegistry: Map[String, List[ConnectorConfig]])
+    (implicit ssc: StreamingContext, transformContext: TransformContext): Option[DStream[FortisItem]] = {
+    import transformContext._
```

Comment:
Why not just pass in a transformContext instance and dereference member variables on there? That could would require much less Scala knowledge and would be more easily parsable for folks from a variety of backgrounds.